doc/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+
+- [ ] The commit message(s) follow our guidelines: https://github.com/dequelabs/axe-core/blob/develop/doc/code-submission-guidelines.md#git-commits
+- [ ] Changes to rules and checks appropriately support [Shadow DOM](https://github.com/dequelabs/axe-core/blob/develop/doc/developer-guide.md)
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
+
+## What is the current behavior?
```

Comment:
We don't need these if we require every PR to have an issue associated with it. You can find the description there.